dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AliasResource.java

```diff
@@ -56,10 +61,11 @@ public AliasResource(SessionFactory sessionFactory, WorkflowResource workflowRes
     @Timed
     @UnitOfWork
     @Path("workflow-versions/{workflowVersionId}")
+    @Operation(operationId = "addAliases", description = "Add aliases linked to a workflow version in Dockstore.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
     @ApiOperation(nickname = "addAliases", value = "Add aliases linked to a workflow version in Dockstore.", authorizations = {
             @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "Aliases are alphanumerical (case-insensitive "
             + "and may contain internal hyphens), given in a comma-delimited list.", response = WorkflowVersion.class)
-    public WorkflowVersion addAliases(@ApiParam(hidden = true) @Auth User user,
+    public WorkflowVersion addAliases(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user", in = ParameterIn.HEADER) @Auth User user,
```

Comment:
`@Parameter(hidden = true)` should be all you need.

Reply:
Yeah, it is the only necessary part but I added the other items because I wanted it to match many of the newer endpoints for consistency.

Reply:
name is fine, but `in = ParameterIn.HEADER` is just wrong.  You can always change the other ones.

Reply:
Maybe elaborate, there is probably something going on with `ParameterIn.DEFAULT` that you're referring to.

Reply:
Removed `in = ParameterIn.HEADER`

Reply:
Was still curious about the explanation here. Also does the swagger-ui "try it now" operation work with `in = ParameterIn.HEADER` missing? Please confirm, thanks!

Reply:
Yeah, tried a few random endpoints and the try it now is still